crates/ledger/src/store.rs

```diff
@@ -1,16 +1,15 @@
 pub mod columns;
-pub mod rocksdb;
 
 use super::kernel::{Epoch, Point, PoolId, TransactionInput, TransactionOutput};
-use crate::ledger::rewards::RewardsSummary;
+pub use crate::rewards::RewardsSummary;
 use columns::*;
 use std::{borrow::BorrowMut, iter};
 
 // Store
 // ----------------------------------------------------------------------------
 
 pub trait Store {
-    type Error;
+    type Error: std::error::Error + Send + Sync + Sized + 'static;
```

Comment:
`Send`, `Sync` and `'static` are required for miette [into_diagnostic](https://docs.rs/miette/latest/miette/trait.IntoDiagnostic.html#tymethod.into_diagnostic).
We might want to relax this constraint somehow.